packages/amazon-pay-integration/src/amazon-pay-v2-payment-strategy.spec.ts

```diff
@@ -0,0 +1,476 @@
+import {
+    AmazonPayV2NewButtonParams,
+    AmazonPayV2PaymentProcessor,
+    AmazonPayV2Placement,
+    getAmazonPayV2,
+    getAmazonPayV2PaymentProcessorMock,
+    getAmazonPayV2Ph4ButtonParamsMock,
+} from '@bigcommerce/checkout-sdk/amazon-pay-utils';
```

Comment:
I think it might be better to move all related to AmazonPayV2 mocks to amazon-pay-integration package, except the case if you are going to use it somewhere else